AlphaWallet/Core/Types/RealmConfiguration.swift

```diff
@@ -0,0 +1,60 @@
+// Copyright SIX DAY LLC. All rights reserved.
+
+import Foundation
+import RealmSwift
+
+struct RealmConfiguration {
+    private static let walletsFolderForTests = "wallets"
```

Comment:
Can you change the value of the string to include "test"? Maybe "testSuiteWallets"?